capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -7,26 +7,51 @@ class BookSearchList extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookList: []
+      bookSearchList: []
     }
-    this.getData = this.getData.bind(this);
   }
 
-  getData() {
+  getData = () => {
     fetch(`/api/search?searchTerm=${this.props.searchQuery}`)
       .then(response => response.json())
-      .then(res => this.setState({ bookList: res }))
+      .then(res => this.setState({ bookSearchList: res }))
       .catch(err => console.log(err));
   }
 
+  getUserBookLists = () => {
+    let userBookLists = [{ id: "1", name: "Best Books" },
+    { id: "2", name: "Fantasy" },
+    { id: "3", name: "Sci-fi" }];
+    return userBookLists;
+  }
+
+  getUserClubs = () => {
+    let userClubs = [{ id: "1", name: "Cool Kats Book Club" },
+    { id: "2", name: "Sci-fi Fans" }];
+    return userClubs;
+  }
+
+  getUserCommunities = () => {
+    let userCommunities = [{ id: "1", name: "Sci-fi Community" },
+    { id: "2", name: "Read-a-lot" }];
+    return userCommunities;
+  }
+
   componentDidMount() {
     this.getData();
+    this.setState({ userBookLists: this.getUserBookLists() });
+    this.setState({ userClubs: this.getUserClubs() });
+    this.setState({ userCommunities: this.getUserCommunities() });
   }
 
   render() {
     var books = [];
-    for (const book of this.state.bookList) {
-      books.push(<BookSearchTile title={book.title} author={book.authors} thumbnailLink={book.thumbnailLink} key={book.id} />);
+    for (const book of this.state.bookSearchList) {
```

Comment:
nit: Rather than building the output list in a loop and then returning it afterward, you could return the result of `map`ping over `bookSearchList` to the `BookSearchTile`s directly here.

Reply:
Good idea! I didn't know it was possible to do it this way until I worked on it in the BookSearchTile